lxc/main_aliases.go

```diff
@@ -62,30 +66,89 @@ func expandAlias(conf *config.Config, args []string) ([]string, bool) {
 	if !foundAlias {
 		aliasKey, aliasValue, foundAlias = findAlias(defaultAliases, origArgs)
 		if !foundAlias {
-			return []string{}, false
+			return []string{}, false, nil
 		}
 	}
 
 	if !strings.HasPrefix(aliasValue[0], "/") {
 		newArgs = append([]string{origArgs[0]}, newArgs...)
 	}
-	hasReplacedArgsVar := false
 
+	// The @ARGS@ are initially any arguments given after the alias key.
+	var ARGS []string
```

Comment:
Please use `args` instead of upper case variables.

Reply:
I was trying to make it clear that this is the value of `@ARGS@`. Though I guess that is clear from the comment. Will change.

Reply:
Renamed to `atArgs` since `args` was already in use.